### PR TITLE
Add whitelabel to solana-embed

### PR DESCRIPTION
**Pull request by shahbaz17**

### CODE REVIEW

File: docs/solana-wallet/api-reference/class.mdx [hunk 1]
```diff
@@ -127,12 +129,119 @@ export interface TorusParams {
...
+   */
+  tncLink?: LocaleLinks<string>;
+  /**
+   * Language specific link for privacy policy on torus-website. See (examples/vue-app) to configure
+   */
+  privacyPolicy?: LocaleLinks<string>;
+  /**
+   * Language specific link for privacy policy on torus-website. See (examples/vue-app) to configure
+   */
+  contactLink?: LocaleLinks<string>;
```
Comment (guru-web3) [inline docs/solana-wallet/api-reference/class.mdx]: not supported as params for now, please remove

Comment (shahbaz17) [reply docs/solana-wallet/api-reference/class.mdx]: ok, let me remove

**Comment by shahbaz17** [comment]
@guru-web3 Updated based on https://github.com/torusresearch/solana-embed/blob/main/src/interfaces.ts#L206

**Review by guru-web3 — APPROVED** [review]
lgtm

**Pull request merged**